group_theory/subgroup.lean

```diff
@@ -107,6 +107,10 @@ namespace is_subgroup
 open is_submonoid
 variables [group α] (s : set α) [is_subgroup s]
 
+@[to_additive is_add_subgroup.coe_neg, simp]
+lemma coe_inv {s : set α} [is_subgroup s] (a : s) : ((a⁻¹ : s) : α) = a⁻¹ := rfl
+attribute [simp] is_add_subgroup.coe_neg
```

Comment:
I think if you put `simp` as the first attribute it will get copied by `to_additive`